blaze/expr/tests/test_arithmetic.py

```diff
@@ -72,3 +72,15 @@ def test_dir():
 def test_arith_ops_promote_dtype():
     r = w + z
     assert r.dshape == dshape('5 * 3 * float64')
+
+
+def test_str_arith():
+    assert isinstance(cs + cs, Concat)
+    assert isinstance(cs * 1, Repeat)
+    assert isinstance(cs % cs, Interp)
+
+    with pytest.raises(Exception):
```

Comment:
It would probably be an `AttributeError` looking up `_div` or something. It would be cool to say that this symbol does not support the '/' operator with a TypeError but as is, this is not the case. I can specialize this to AttributeError though.